tests/Agent/UnitTests/Core.UnitTest/Database/SqlParsingCrossAgentTest.cs

```diff
@@ -18,18 +18,21 @@ namespace NewRelic.Agent.Core.NewRelic.Agent.Core.Database
     [TestFixture]
     public class SqlParsingCrossAgentTest
     {
-        [TestCaseSource("GetSqlParsingTestCases")]
+        [TestCaseSource(nameof(GetSqlParsingTestCases))]
         public void SqlParsingTest(string inputSql, string expectedOperation, string expectedTable)
         {
             var parsed = SqlParser.GetParsedDatabaseStatement(DatastoreVendor.MSSQL, CommandType.Text, inputSql);
-            Assert.AreEqual(expectedOperation.ToLower(), parsed.Operation, string.Format("Expected operation {0} but was {1}", expectedOperation, parsed.Operation));
-            Assert.AreEqual(expectedTable?.ToLower(), parsed.Model, string.Format("Expected table {0} but was {1}", expectedTable, parsed.Model));
+            Assert.Multiple(() =>
+            {
+                Assert.That(parsed.Operation, Is.EqualTo(expectedOperation.ToLower()), string.Format("Expected operation {0} but was {1}", expectedOperation, parsed.Operation));
+                Assert.That(parsed.Model, Is.EqualTo(expectedTable?.ToLower()), string.Format("Expected table {0} but was {1}", expectedTable, parsed.Model));
+            });
         }
 
 
         // You can uncomment this to measure timing of sql parsing
         //[Test]
-        public void Timing()
+        private void Timing()
```

Comment:
Instead of making this private, should this just be removed? It looks like it's an "test" that was used to time things.

Reply:
Done.